### PR TITLE
[New] Trigyn testcases record actions

### PR DESCRIPTION
**Pull request by Srinivas9933**
[New] Added test cases for record actions.

### CODE REVIEW

File: app/javascript/components/record-actions/container.spec.js [hunk 1]
```diff
@@ -135,7 +125,7 @@ describe("<RecordActions />", () => {
       })
     })
   };
-  let component;
+  //   let component;
```
Comment (aespinoza-quoin) [inline app/javascript/components/record-actions/container.spec.js:128]: remove this comment

File: app/javascript/components/record-actions/container.spec.js [hunk 2]
```diff
@@ -208,91 +197,54 @@ describe("<RecordActions />", () => {
...
-        "mode",
-        "clearSelectedRecords"
-      ].forEach(property => {
-        expect(transitionsProps).to.have.property(property);
-        delete transitionsProps[property];
-      });
-      expect(transitionsProps).to.be.empty;
+      mountedComponent(<RecordActions {...props} />, defaultStateWithDialog(TRANSFER_DIALOG));
+
+      expect(screen.queryAllByRole("dialog")).toHaveLength(1);
```
Comment (aespinoza-quoin) [inline app/javascript/components/record-actions/container.spec.js:231]: Verify if it is a Transfer

File: app/javascript/components/record-actions/container.spec.js [hunk 3]
```diff
@@ -208,91 +197,54 @@ describe("<RecordActions />", () => {
...
   describe("Component RequestApproval", () => {
-    beforeEach(() => {
-      ({ component } = setupMountedComponent(RecordActions, props, defaultStateWithDialog(REQUEST_APPROVAL_DIALOG)));
-    });
-
     it("renders RequestApproval", () => {
-      expect(component.find(RequestApproval)).to.have.length(1);
+      mountedComponent(<RecordActions {...props} />, defaultStateWithDialog(REQUEST_APPROVAL_DIALOG));
+
+      expect(screen.queryAllByRole("dialog")).toHaveLength(1);
```
Comment (aespinoza-quoin) [inline app/javascript/components/record-actions/container.spec.js:223]: Verify if it is a RequestApproval

**Review by aespinoza-quoin — CHANGES_REQUESTED** [review]

File: app/javascript/components/record-actions/container.spec.js (hunk 1 above)
Comment (Srinivas9933) [reply app/javascript/components/record-actions/container.spec.js]: Done

File: app/javascript/components/record-actions/container.spec.js (hunk 2 above)
Comment (Srinivas9933) [reply app/javascript/components/record-actions/container.spec.js]: Done

File: app/javascript/components/record-actions/container.spec.js (hunk 3 above)
Comment (Srinivas9933) [reply app/javascript/components/record-actions/container.spec.js]: Done

**Pull request merged**